examples/sn-react-browser/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@sensenet/react-browser",
+  "version": "1.0.2",
+  "private": true,
+  "author": "Sense/Net <snteam@sensenet.com> (http://sensenet.com/)",
+  "contributors": [
+    "Dániel Blaskó <blaskoi@gmail.com>"
+  ],
+  "license": "GPL-2.0",
+  "homepage": "https://sensenet.com",
+  "keywords": [
+    "sensenet",
+    "cps",
+    "single page application",
+    "react",
+    "typescript"
+  ],
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "eject": "react-scripts eject"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.0.23",
+    "@types/node": "^12.12.14",
+    "@types/react": "^16.9.13",
+    "@types/react-dom": "^16.9.4",
+    "@types/react-router": "^5.1.2",
+    "@types/react-router-dom": "^5.1.2",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-scripts": "^3.2.0",
+    "typescript": "^3.7.2"
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.7.0",
```

```suggestion
    "@material-ui/core": "^4.6.1",
```
please change this back to 4.6.1 we will upgrade material ui after the release. Also don't forget to update the yarn.lock